tests/cisco_ios/show_vrf/cisco_ios_show_vrf.raw

```diff
@@ -167,3 +167,6 @@
                                                                    Vl940
                                                                    Vl941
                                                                    Vl946
+  vpn7                             127.0.0.1:7         ipv4        Vl5
```

Comment:
Can you please upload a separate **.raw** and **.parsed**? This helps ensure backwards capability and adds more data to test the template against.

Reply:
I extracted the raw and parsed files into new one. Hope it's what you expected ;)